src/pages/404.html

```diff
@@ -10,12 +10,22 @@
   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap" rel="stylesheet" crossorigin="anonymous">
   <link href="styles/common.css" rel="stylesheet" type="text/css">
   <link href="styles/404.css" rel="stylesheet" type="text/css">
+  <script src="https://kit.fontawesome.com/6cd31f02df.js" crossorigin="anonymous"></script>
+  <script type="module" defer src="scripts/404.js"></script>
 </head>
 <body>
 
   <header>
     <img src="assets/back.svg" alt="Go back" onclick="window.history.back()">
     <h1>Not found</h1>
+    <nav id="top-menu">
+      <button id="menuButton" type="button"><i class="fas fa-bars"></i></button>
```

Comment:
Would it be possible to just have `<nav id="top-nav"></nav>` in the html files and render the rest from the `renderTopMenu.js` script with the help of the `ElementHelper` class? It could be nice if we don't need to have a lot of code repetition. There actually has been a small change to `ElementHelper` recently that makes it simpler to use. The `setParent` now accepts both other `ElementHelper` instances as well as regular old `HtmlElements` (what you get for example from `document.getElementById()`), so you could do something like this:
```js
const navbar = document.getElementById('top-menu');
const button = ElementHelper.create('button').setId('id').setParent(navbar);
ElementHelper.create('i').setClass('fas fa-bars').setParent(button);
```